requirements.txt

```diff
@@ -0,0 +1,11 @@
+gunicorn==23.0.0
+mysqlclient==2.2.6
+
+#Django
+Django==5.1.4
+django-mysql==4.15.0
+django-webpack-loader==3.1.1  
+
+debugpy==1.8.8
```

Comment:
This should be updated to 1.8.11, also might want to check the versions of the other dependencies here?

Reply:
ok

Reply:
Done